packages/core/src/___internal-do-not-use-will-break-in-patch/admin-ui/pages/ListPage/index.tsx

```diff
@@ -248,6 +248,7 @@ const ListPage = ({ listKey }: ListPageProps) => {
         'Error...'
       ) : data && metaQuery.data ? (
         <Fragment>
+          {list.description !== null && <p>{list.description}</p>}
```

Comment:
24px top margin, and 704px max width please. Otherwise great! 🤚 

![image](https://user-images.githubusercontent.com/6447754/168540323-d1bc2435-cec9-4889-b02c-dee80d341657.png)

Reply:
Applied the above stylus changes in https://github.com/keystonejs/keystone/pull/7537/commits/479624a28e4fc4f7550a64edf0920f73105b6527 commit